### PR TITLE
add finance subteam

### PR DESCRIPTION
**Pull request by CJ-Wright**
Please add yourself if you are interested in being part of the finance subteam

### CODE REVIEW

**Comment by CJ-Wright** [comment]
Note that this will require a vote at some point

**Comment by CJ-Wright** [comment]
@conda-forge/core
This PR falls under the Sub-team Formation policy, please vote and/or comment on this PR.
This PR needs 50% of core to vote yea to pass.
To vote please leave Approve (yea) or Request Changes (nay) reviews.
If you would like changes to the current language please leave a comment or push to this branch.
This vote will end on July 20th 2020 at 6pm Eastern.

File: src/orga/subteams.rst [hunk 1]
```diff
@@ -45,6 +45,41 @@ Members
...
+
+- keep core aware of budgetary and financial matters pertaining to conda-forge
+- facilitate dispersal of funds
+- give core updates via the standing budget item at core meetings
+- work closely with the NumFOCUS point of contact to ensure smooth financial operations
+
+This team will operate via the following rules
+
+- only members of ``core`` can be on this sub-team
+- this sub-team will report its activity to core at all ``core`` meetings
```
Comment (mbargull) [inline src/orga/subteams.rst:74]:
```suggestion
- this sub-team will report its activity to core at all attended ``core`` meetings
```
or something. Meaning, it's fine if (currently the both of) you are on vacation at the same time and no one reports ;).

Comment (CJ-Wright) [reply src/orga/subteams.rst:74]: I was kind of hoping that we'd have enough members to cover the meetings, but I guess not.

Comment (mbargull) [reply src/orga/subteams.rst]: (To be fair, the listed members are probably the most consistently attending -- I really can't remember a meeting in which not at least one of you was there!)

**Comment by CJ-Wright** [comment]
I think this passes. I'll go make the needed GH teams

**Pull request merged**